andela/pippin-ah-backend
# #162727543 Configure eslint

#### What does this PR do?
This PR adds and configures eslint for the API

#### Description of Task to be completed?
- Add eslint dependency to package.json
- Add .eslint file to the root directory
- Add rules for eslint
- Add .eslintignore file to the root directory
- Add default files to .eslintignore

#### How should this be manually tested?
To manually test this:
- Clone this repository
- Install eslint in your code editor
- Add the code: `var: newVariable = 'randomStatement'` to index.js;
- Eslint should give the error: 'Unexpected var, use let or const instead.eslint(no-var)'

#### What are the relevant pivotal tracker stories?
The relevant pivotal tracker story for this PR:
 - #162727543 https://www.pivotaltracker.com/n/projects/2229853/stories/162727543/
